main/src/ca/uwaterloo/flix/api/lsp/Indexer.scala

```diff
@@ -19,6 +19,7 @@ import ca.uwaterloo.flix.api.lsp.Index.traverse
 import ca.uwaterloo.flix.language.ast.TypedAst.Predicate.{Body, Head}
 import ca.uwaterloo.flix.language.ast.TypedAst.*
 import ca.uwaterloo.flix.language.ast.*
+import ca.uwaterloo.flix.language.ast.shared.SymUse.{AssocTypeSymUse, CaseSymUse, DefSymUse, LocalDefSymUse, RestrictableCaseSymUse, SigSymUse}
```

Comment:
I am surprised Intellij IDEA does not collapse this into a wildcard import.

Is it because its inside an object or what?

Reply:
It does, I just didn't format it

Reply:
This is messed up to do after the fact. Can we let it stay and do a separate PR where we just run the import optimizer on the whole project?

Reply:
Running the import optimizer project-wide is probably too much. But I suppose we can leave your imports as-is. I was just curious.

Reply:
If it is less than 500 lines diff, we can do it in a separate PR later.